### DISCUSSION TITLE
How can I go back to Dreambooth version of ~2 weeks old?

### DISCUSSION THREAD

**Discussion by Renaldas111** [d8ahazard/sd_dreambooth_extension#311] [Q&A]
I was using Dreambooth with almost no problems up to last huge update, and it is totally broken. I want to go back to the version I used before, but being too stupid with all this github stuff I cant understand what and how can I do this.
I checked for updates in Extension tab in A1111 webui, updated the Dreambooth from there, and all is totally broken. Tons of new functions in new places is also a nightmare :(
Just want to go back to a version I used several weeks ago.

**Comment by mykeehu** [comment]
- Open a cmd
- Go to dreambooth folder
- type this: `git reset --hard` (and short commit number, like efb7064) and enter
- start SD :)
